README.md

```diff
@@ -1 +1,2 @@
-# Code-review-repo
\ No newline at end of file
+# Code-review-repo
+In this repo, I will be reviewing my own code. However, your contributions are highly welcomed!
```

Comment:
Kindly add a title above to your readme file. This will allow our repo users to understand the kind of project it is.